modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewChangesDisplayContext.java

```diff
@@ -1469,13 +1469,20 @@ else if (_log.isDebugEnabled()) {
 					int changeType = _ctDisplayRendererRegistry.getChangeType(
 						ctEntry, model);
 
-					if (_ctDisplayRendererRegistry.isWorkflowEnabled(
```

Comment:
I thought I needed it so that the workflow tab could be visible while the workflow actions menu would be hidden for an already-published Publication. I'll try putting it back or looking into another solution.

Reply:
Wouldn't this check be sufficient `_ctCollection.getStatus() == WorkflowConstants.STATUS_DRAFT` ?

Plus, we would still need to check whether workflow is enabled before adding the workflow tab.

Reply:
The check you suggested did the trick. Pushed the updated fix, thanks!